goserver/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/google/meet-on-fhir/server"
+)
+
+var authorizedFhirEndpoint = flag.String("authorized_fhir_endpoint", "", "The FHIR endpoint that is authorized to laucnh the telehealth app")
```

Comment:
Explain what happens if this flag isn't set. We should probably return an error if this empty because otherwise they'll just get a more confusing error later on. Adding this validation in server.go would probably make more sense so we can unit test it more easily.

Reply:
Done.